### DISCUSSION TITLE
How To Access Node UI Outside Of My House?

### DISCUSSION THREAD

**Discussion by andersonkoh96** [mysteriumnetwork/node#4983] [Q&A]
Hi, I have 2 questions.

1. Do I need to allow any port in my Pi or router?

2. I want to connect my Node UI outside of my house and use Zerotier. So everything should be only accessible by me, but I'm unable to reach my Node UI with port 4449. instead, I'm able to ping my Pi or even SSH it. I tested and kept getting connection refused with the command `nc -vz xx.xx.xx. 4449`.

Please see the picture below. 8966 is my Pi SSH port.

<img width="520" alt="image" src="https://user-images.githubusercontent.com/43630453/162176099-a445df16-d5cb-43ce-b595-343baec5b47a.png">

**Comment by adinetech** [comment]
Forward 4449 TCP port or use tailscale, you can also set up a private VPN server.
